app/dose-event-slider/dose-event-slider.component.ts

```diff
@@ -17,13 +18,42 @@ export class DoseEventSliderComponent {
     @Input()
     public doseEvent: DoseEvent;
 
+    constructor(
+        private doseEventService: DoseEventService
+    ) {
+    }
+
+    saveChanges() {
+        /*
+         * Ignore server response, just assume it worked.
+         * There is also a timing issue here if the user quickly does multiple actions
+         * and they get out of order in the network.
+         */
+        this.doseEventService.saveList(
+            [
+                this.doseEvent
+            ]
+        );
```

Comment:
Marking a DoseEvent as "TAKEN", "SKIPPED", or neither (by deleting the action) now saves the change to the server.

Because we don't have a save button on this screen, and it would probably be a poor UI to put up a loading bar every time you mark a dose as taken, there is a potential timing issue here where multiple user actions could get out of order in the network and the last action is not the one we saved.

If this is an issue, the easiest fix would be to not save on any changes in this screen, and have a save button, but don't know if we want that.